codice/ddf
# DDF-4994 Remove Embedded Solr Provider

#### What does this PR do?
Removes the no longer required EmbeddedSolrProvider.

#### Who is reviewing it? 
@lamhuy 
@rzwiefel 

#### Select relevant component teams: 
@codice/solr 

#### Ask 2 committers to review/merge the PR and tag them here.
@ahoffer
@bdeining 
@paouelle
@pklinef

#### How should this be tested?
Full build with tests.
Unzip and Run and verify basic functionality works as expected (Create workspace, etc)

#### Any background context you want to provide?
The Embedded Solr Provider is no longer required by any code, so removing this to clean up baseline.

#### What are the relevant tickets?
Fixes: #4994 

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
